core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -50,6 +50,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.iceberg.MetadataTableUtils.createMetadataTableInstance;
```

Comment:
We avoid static method imports. Can you call `MetadataTableUtils.createMetadataTableInstance` instead?

Reply:
Ah OK good to know. I'll fix the other places in the PR as well.